Xamarin.Forms.Platform.UAP/FormsTextBox.cs

```diff
@@ -461,7 +461,7 @@ public static Size GetCopyOfSize(FormsTextBox control, Windows.Foundation.Size c
 			{
 				_copyOfTextBox = new FormsTextBox
 				{
-					Style = Windows.UI.Xaml.Application.Current.Resources["FormsTextBoxStyle"] as Windows.UI.Xaml.Style
+					Style = Windows.UI.Xaml.Application.Current.Resources["FormsTextBoxStyleMeasure"] as Windows.UI.Xaml.Style
```

Comment:
@hartez Here is the only place the Measure template is used, in that single instance used calc the size off the visual tree.  Which was the fix for #2172